runtime/vm/JFRChunkWriter.hpp

```diff
@@ -172,6 +173,7 @@ class VM_JFRChunkWriter {
 	static constexpr int CLASS_LOADING_STATISTICS_EVENT_SIZE = 5 * sizeof(I_64);
 	static constexpr int THREAD_CONTEXT_SWITCH_RATE_SIZE = sizeof(float) + (3 * sizeof(I_64));
 	static constexpr int THREAD_STATISTICS_EVENT_SIZE = (6 * sizeof(U_64)) + sizeof(U_32);
+	static constexpr int THREAD_DUMP_EVENT_SIZE = 10000;
```

Comment:
I dont think this is large enough if you have many threads. I think it should be relative to the number of threads.